Build/templates/run_tests.yml

```diff
@@ -91,6 +91,21 @@ steps:
       failOnStderr: true
       pythonInterpreter: $(PYTHON_INTERPRETER)
 
+  # Setup the glass test folder but use the cache instead of downloading
+  # skip --getGlass
+  - task: PythonScript@0
+    displayName: 'Setup glass test folder'
+    condition: and(succeeded(), eq(variables['CACHE_RESTORED'], 'true'))
+    env:
+      SYSTEM_ACCESSTOKEN: $(DropToken)
+    inputs:
+      scriptSource: 'filePath' # 'filePath' | 'inline'. Required. Script source. Default: filePath.
+      scriptPath: $(Build.SourcesDirectory)/Build/setup_glass.py
+      arguments: '--getDropExe --copyGlass --getVsTestConsole --copyPtvsOutput --verifyListing --authTokenVariable SYSTEM_ACCESSTOKEN --buildOutput $(Build.BinariesDirectory)\raw\binaries'
```

Comment:
I still don't think this is correct. 

If you're pulling from the glass tests folder, you only need to do the run_glass script. Everything else should be in the cache. 

You can skip this entire step.

Reply:
ok. nice. then i dont need to modify setup_glass.py